### ISSUE TITLE
Running 2 LOAD DATA LOCAL INFILE queries in one query() call yields PARSER_READ_PAST_END

### ISSUE THREAD

**Issue by NewsteadEric** [felixge/node-mysql#1036]
Hello,

When trying to run the following two queries in the same ```query()``` call
```
LOAD DATA LOCAL INFILE 'file_name_here'
    INTO TABLE table_a
    CHARACTER SET utf8
    FIELDS TERMINATED BY ','
    OPTIONALLY ENCLOSED BY '"'
    LINES TERMINATED BY '\n'
    IGNORE 1 LINES;

LOAD DATA LOCAL INFILE 'file_name_here'
   INTO TABLE table_b
   CHARACTER SET utf8
   FIELDS TERMINATED BY ','
   OPTIONALLY ENCLOSED BY '"'
   LINES TERMINATED BY '\n'
   IGNORE 1 LINES;
```

node-mysql returns the error message ```{ [Error: Parser: read past end] offset: 160, code: 'PARSER_READ_PAST_END' }```

Running each of the queries in its own query() call works fine, however.

What could be causing this, and is there any way to get this to work without having to separate the two queries from each other in terms of calling node-mysql?

**Comment by dougwilson** [comment]
Well, it sounds like a bug. We'll investigate.

**Comment by dougwilson** [comment]
I have verified the bug indeed exists.

**Comment by sidorares** [comment]
@dougwilson seems to work with node-mysql2 Might look into this as well ( no promise though, short on time )

**Comment by dougwilson** [comment]
Thanks @sidorares , I have fixed the issue for the 2.6 release tonight :)

**Comment by sidorares** [comment]
that was quick :)
